ADA/ada.qmd

```diff
@@ -21,13 +21,16 @@ It brings together:
 
 As long as you're connected to the network - it's available to anyone who works for the Department of Education. 
 
-The ADA project is currently in its onboarding phase, which is expected to continue until 2025. At the moment, teams can onboard voluntarily until **???**. Following this date, the remaining teams will have their data migrated for them. 
+The ADA project is currently in its onboarding phase.  The ADA team will work with analyst teams to plan migrations. We encourage teams to engage early so that you have time to migrate your work and receive support from the project team. 
```

Comment:
Minor thing - looks like you have a double space after the first sentence in this paragraph